subsys/net/l2/wifi/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config WIFI_FEAT_RAW_SCAN_RESULTS
```

Comment:
Not sure I follow why is there `FEAT` in the name. As it's wifi_mgmt configuration, I'd call this `WIFI_MGMT_RAW_SCAN_RESULTS`